app/src/main/java/org/luxoft/sdl_core/AndroidSettings.java

```diff
@@ -0,0 +1,89 @@
+package org.luxoft.sdl_core;
+
+import android.util.Log;
+
+import java.util.HashMap;
+
+public class AndroidSettings {
+
+    private static final String AndroidSection = "ANDROID";
+    private static final String TAG = AndroidSettings.class.getSimpleName();
+
+    private static HashMap<String, String> mValues;
+
+    public enum IniParams {
+        BUFFER_SIZE("BufferSize"),
+        READER_SOCKET_ADDRESS("ReaderSocketAdress"),
+        CONTROL_SOCKET_ADRESS("ControlSocketAdress"),
+        WRITER_SOCKET_ADRESS("WriterSocketAdress"),
+        PREFFER_MTU("PrefferredMtu"),
+        SDL_TESTER_SERVICE_UUID("SdlTesterServiceUUID"),
+        MOBILE_NOTIFICATION_CHARACTERISTIC("MobileNotificationCharacteristic"),
+        MOBILE_RESPONCE_CHARACTERISTIC("MobileResponceCharacteristic");
+
+        public final String param;
+
+        IniParams(String param) {
+            this.param = param;
+        }
+    }
+
+    private static String getStrValue(IniLoader iniFile, String section, String Key, String defaultValue) {
+        String result = defaultValue;
+
+        if(iniFile.containsKey(section, Key)) {
+            result = iniFile.getValue(section, Key).trim().replaceAll("^\"+|\"+$", "");
+        }
+
+        return result;
+    }
+
+    public static  void loadSettings(String filepath) {
+
+        HashMap<String, String> defaulValues  = new HashMap<String, String>() {{
+            put("BufferSize",                       "131072");
+            put("ReaderSocketAdress",               "./localBleReader");
+            put("ControlSocketAdress",              "./localBleControl");
+            put("WriterSocketAdress",               "./localBleWriter");
+            put("PrefferredMtu",                    "512");
+            put("SdlTesterServiceUUID",             "00001101-0000-1000-8000-00805f9b34fb");
+            put("MobileNotificationCharacteristic", "00001102-0000-1000-8000-00805f9b34fb");
+            put("MobileResponceCharacteristic",     "00001104-0000-1000-8000-00805f9b34fb");
+        }};
```

Comment:
I think we can use value of enum (IniParams) for key. 
`put(IniParams.BUFFER_SIZE.param, "131072");`